pmm/users.tf

```
@@ -0,0 +1,27 @@
+# TODO we need to use roles instead users
+resource "aws_iam_user" "nikita" {
```

```suggestion
resource "aws_iam_user" "nbeletskii" {
```
Let's maybe adopt a naming convention, so that every new user is composed of the first letter of their name plus their family name.